jmui/jmui/open_wallet_dialog.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file is generated based on `open_wallet_dialog.ui`, but the python format is much better for the code review process than the original `.ui` file.